src/Backend/Modules/Location/Command/CopyLocationWidgetsToOtherLocaleHandler.php

```diff
@@ -15,8 +17,12 @@ public function __construct(SpoonDatabase $database)
         $this->database = $database;
     }
 
-    public function handle(CopyLocationWidgetsToOtherLocale $copyLocationWidgetsToOtherLocale): void
+    public function handle(CopyModuleToOtherLocaleCommandInterface $command): void
     {
+        if (!$command instanceof CopyLocationWidgetsToOtherLocale) {
+            throw new \Exception('The class should be ' . CopyLocationWidgetsToOtherLocale::class);
```

Comment:
This description could be clearer. Also; consider creating a custom exception or at least use a `use` statement

Reply:
Custom exception added